### PR TITLE
IDE-860 Autocomplete doesn't for builder windows

### PR DESCRIPTION
**Pull request by dehilsterlexis**

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by richardkchapman** [comment]
@dehilsterlexis Please rebase to 7.2.x

File: comms/EclMeta.cpp [hunk 1]
```diff
@@ -315,6 +318,13 @@ void CEclMeta::PopulateMeta(const boost::filesystem::wpath & fileOrDir, const st
     }
 }
 
+CEclFile *CEclMeta::CreateFile(const std::_tstring & dottedPath, const std::_tstring & path)
+{
+    CEclFile *file = new CEclFile(path);
+    m_masterMeta[dottedPath] = file;
+    return file;
+}
```
Comment (GordonSmith) [inline comms/EclMeta.cpp]: Need to ensure that this is placed into a link counted container so it doesn't leak?

Comment (GordonSmith) [reply comms/EclMeta.cpp]: (just checked and it is.)

Comment (dehilsterlexis) [reply comms/EclMeta.cpp]: @GordonSmith yes. You are right.

**Pull request merged**